app/src/main/java/org/mifos/selfserviceapp/presenters/LoginPresenter.java

```diff
@@ -147,7 +148,14 @@ public void onNext(Page<Client> clientPage) {
                         getMvpView().hideProgress();
                         if (clientPage.getPageItems().size() != 0) {
                             long clientId = clientPage.getPageItems().get(0).getId();
+                            Context c = context.getApplicationContext();
+                            int mode = Context.MODE_PRIVATE;
+                            String spName = Constants.SHAREDPREF_NAME;
+                            SharedPreferences sp = c.getSharedPreferences(spName, mode);
                             getMvpView().showClient(clientId);
+                            String keyClientId = Constants.SP_KEY_CLIENTID;
+                            sp.edit().putLong(keyClientId, clientId);
+                            sp.edit().commit();
```

Comment:
@therajanmaurya 

1.There is no such file named PreManager.class under util. Are you talking about api/local/PreferencesHelper?

2. **If you are talking about api/local/PreferencesHelper**, then what is the need for change ?